src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -2112,6 +2112,18 @@ describe("Artwork type", () => {
       })
     })
 
+    it("is set to calculated at checkout when Arta shipping", () => {
+      artwork.arta_enabled = true
+
+      return runQuery(query, context).then((data) => {
+        expect(data).toEqual({
+          artwork: {
+            shippingInfo: "Shipping: Calculated in checkout",
```

Comment:
I assume that was approved by Joh. Not an expert here but `IN` seems like a strange thing to use here. But what do I know:)

Reply:
I'm also not an expert 🙂. I took latest variant from JIRA.